index.d.ts

```diff
@@ -0,0 +1,471 @@
+// Type definitions for realm-js 1.8.0
```

Comment:
Is it necessary to include the version number in this file? It seems likely to get out of sync unless it's automatically updated as part of the release process.